src/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScanner.java

```diff
@@ -3,7 +3,7 @@
  *
  * ZAP is an HTTP/HTTPS proxy for assessing web application security.
  *
- * Copyright 2012 The ZAP development team
+ * Copyright 2017 The ZAP development team
```

Comment:
We don't update the copyright years on existing classes (not that I want to start a copyright year war, just noting ;)